tests/spec/core/webidl-spec.js

```diff
@@ -494,13 +494,13 @@ describe("Core - WebIDL", () => {
   // 5
   getter float ();
   // 6
-  getter float withName();
+  getter float withName ();
   // 7
   setter void ();
   // 8
-  setter void named();
+  setter void named ();
   // 9
-  static Promise<RTCCertificate> generateCertificate(AlgorithmIdentifier keygenAlgorithm);
+  static Promise<RTCCertificate>  generateCertificate(AlgorithmIdentifier keygenAlgorithm);
```

Comment:
Any reason this one generates an extra space?

Reply:
Because the original HTML has the space. Happy to see the migration fixes this case.